lib/Controller/PollController.php

```diff
@@ -126,13 +127,85 @@ public function createPoll(string $question, array $options, int $resultMode, in
 
 		try {
 			$this->chatManager->addSystemMessage($this->room, $attendee->getActorType(), $attendee->getActorId(), $message, $this->timeFactory->getDateTime(), true);
-		} catch (\Exception $e) {
+		} catch (Exception $e) {
```

Comment:
Personal preference, but I'm against importing base exceptions, to make it easily and explicit visible that a base exception is used.